Arduino-Bling/RobotBling/animations.ino

```diff
@@ -443,3 +443,60 @@ void flash(unsigned long timeInterval, uint32_t color, uint8_t brightness)
   setAllPixelsOff();
   showPixels();
 }
+
+void rainbowCycle(int SpeedDelay) {
```

Comment:
As a note, `delay()` function in Arduino is an `unsigned long`. Regardless, you don't want an accidental signed argument pass to this function, it will result in an unpredictable delay.